eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -76,14 +76,14 @@ library
         cardano-crypto-class,
         cardano-crypto-test,
         cardano-crypto-wrapper,
-        cardano-data,
-        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.2,
+        cardano-data ^>=1.2,
+        cardano-ledger-binary:{cardano-ledger-binary, testlib} ^>=1.3,
         cardano-ledger-byron,
         cardano-ledger-byron-test,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.9,
+        cardano-ledger-core:{cardano-ledger-core, testlib} ^>=1.10,
         cardano-ledger-pretty,
-        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} >=1.6 && <1.10,
-        cardano-protocol-tpraos:{cardano-protocol-tpraos, testlib} >=1.0.1,
+        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} ^>=1.9,
+        cardano-protocol-tpraos:{cardano-protocol-tpraos, testlib} ^>=1.0.1,
```

It was intentional. We don't want upper bound on `tpraos`
```suggestion
        cardano-protocol-tpraos:{cardano-protocol-tpraos, testlib} >=1.0.1,
```